results-processor/README.md

```diff
@@ -26,3 +26,28 @@ virtualenv itself).
 deactivate  # or in a different shell
 tox
 ```
+
+## Managing dependencies
+
+We pin all dependencies recursively in `requirements.txt`, and use
+[`pyup`](../.pyup.yml) to automatically upgrade them. However,
+`requirements.txt` is a flat list and does not differentiate direct
+dependencies from transitive ones, so when a direct dependency drops a sub
```

Comment:
If this is a problem, how about using requirements.in as in https://github.com/foolip/ecosystem-infra-rotation/blob/master/requirements.in? Doesn't pyup support that?